javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -236,11 +235,12 @@ public Object slang_persistent() {
         }
     }
 
+    /** Aggregated, randomized access to every element */
     public static class Get extends Base {
         @Benchmark
         public int java_mutable() {
             int aggregate = 0;
-            for (int i = 0; i < CONTAINER_SIZE; i++) {
+            for (int i : RANDOMIZED_INDICES) {
```

Comment:
`Scala` optimizes for sequential access (i.e. it extracts the most recently used leaf node, providing actual constant time access to it). I personally find it weird that an immutable data structure stores part of its history...

What we want is to simulate random access, hence the iteration.